source-map.bs

```diff
@@ -197,7 +196,74 @@ IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.</p>
 Scope {#scope}
 ==============
 
-This Standard defines the Source Map Format.
+This Standard defines the source map format, used by different types of
+developer tools to improve the debugging experience of code compiled to
+JavaScript, WebAssembly, and CSS.
+
+Conformance {#conformance}
+==========================
+
+A conforming source map document is a JSON document that conforms to the
+structure detailed in this specification.
+
+A conforming source map generator should generate documents which are
+conforming source map documents, and can be decoded by the algorithms in this
+specification without reporting any errors (even those which are specified as
+optional).
+
+A conforming source map consumer should implement the algorithms specified in this
+specification for retrieving (where applicable) and decoding source map documents.
+A conforming consumer is permitted to accept non-conforming source map documents,
+and is permitted to ignore errors or report them without terminating where the
+specification indicates that an algorithm may [=optionally report an error=].
```

Comment:
```suggestion
A conforming consumer is permitted to ignore errors or report them without terminating where the specification indicates that an algorithm may [=optionally report an error=].
```

I would avoid mentioning non-conforming source map documents, since the goal is to eventually encode all the "error recovery" cases as optional errors in the spec itself.